src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/ThrowHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Microsoft.Data.SqlClientX
```

Comment:
Doesn't ADP aka AdapterUtil class already provide these convenience throw helpers ?

Reply:
yes, deleted this file. Ben had a similar comment.